### ISSUE TITLE
More color / icon alternatives for the Calendar

### ISSUE THREAD

**Issue by stevejburge** [publishpress/PublishPress#346]
Maybe we can make the Calendar display options more flexible? At the moment, both colors and icons are used for the same thing (statuses)

Thanks to @Jorr :

> Maybe it has become clear from the text above that our website uses
> CPT pretty heavily. So for our editors *it would be great if different
> post types could be visually distinguished on both the calendar and the
> content overview* screens. I would sacrifice color-coded statuses for
> color-coded post types. I could use different opacity for statuses or just
> rely on the icons (and make them more pronounced). The Content Overview
> itself can only group by category (it would be great to also be able to
> group by post types). In fact, I will probably create some sort of
> customization on my end for this as well, but as you know merging
> customizations with updates is a great bother :)

**Comment by stevejburge** [comment]
>  I wish there was an option to change the color scheme of the calendar, so that instead of displaying background color based on post status, you could choose to have color indicate author and/or category.

**Comment by andergmartins** [comment]
Makes sense the argument that color and icons are used for the same thing, but there is a good thing on that which is related to accessibility to color blindness or similar disability or special needs. Do you think the icon is enough for color blind people, or do we need a color blind mode option on the user profile for accessibility? 

But it makes sense as well that we could provide options to customize the color pallet by other attributes then status.

**Comment by stevejburge** [comment]
One more:
https://wordpress.org/support/topic/feature-request-post-color-by-author-category-or-status/

> I’ve seen that I’m not the first person wishing there was the possibility to color-code the articles based on categories:
> https://wordpress.org/support/topic/feature-request-post-color-by-author-category-or-status/
> 
> Just was wondering if there is an update on that front if this feature is at the top of the roadmap?
